gtsam/gtsam.i

```diff
@@ -597,6 +597,7 @@ class Rot3 {
   Rot3(double R11, double R12, double R13,
       double R21, double R22, double R23,
       double R31, double R32, double R33);
+  Rot3(double w, double x, double y, double z);
```

Comment:
Is this related to this PR? (I'm assuming it's a quaternion rot constructor?)

Reply:
It isn't, but it was minor and I didn't want to create a whole new PR for it.